### ISSUE TITLE
How to resolve `unsafe-eval` issues for extension development

### ISSUE THREAD

**Issue by jonahlibrach** [reduxjs/redux-devtools#1048]
I'm getting some 
![image](https://user-images.githubusercontent.com/9609550/150665529-4fbab168-d392-44ab-9f25-56e8903e4bcf.png)

How do I resolve these issues when building the app with webpack?
Is it a simple modification to the `dev.config.babel.js`? 

Albeit unsuccessfully, I tried to follow what was recommended here: https://stackoverflow.com/questions/48047150/chrome-extension-compiled-by-webpack-throws-unsafe-eval-error

Has anyone encountered this before? How to move forward and get it working?

**Comment by Methuselah96** [comment]
Try un-commenting [this line](https://github.com/reduxjs/redux-devtools/blob/main/extension/webpack/base.config.js#L10) in the base config. I don't have time to check if that works right now, but I'll take a look later.

**Comment by jonahlibrach** [comment]
Doing that caused this:
![image](https://user-images.githubusercontent.com/9609550/150697979-fcbcfbd9-54c6-4947-9d93-3a6b8579b68f.png)

**Comment by hecjhs** [comment]
@Methuselah96 did you have any chance to check what's going on, I'm getting the same error

**Comment by Methuselah96** [comment]
Is this causing the DevTools to not work? I get the first error when I go to GitHub with the dev build, but the DevTools still seem to work for sites that have the extension enabled.

**Comment by Methuselah96** [comment]
@jonahlibrach Also, just to confirm, are you using Yarn? This repository only works with Yarn.

**Comment by Methuselah96** [comment]
@jonahlibrach @hecjhs Were you able to check whether this was causing the DevTools to not work?

**Comment by Methuselah96** [comment]
I'm going to go ahead and close this, but feel free to reply if you have any new information to help reproduce this issue and I can reopen it.

**Comment by hecjhs** [comment]
@Methuselah96 I'm still having issues with this, can you provide documentation on how I should run this on dev mode?

**Comment by Methuselah96** [comment]
Is this causing the DevTools to not work? I get the first error when I go to GitHub with the dev build, but the DevTools still seem to work for sites that have the extension enabled.

**Comment by hecjhs** [comment]
yes, when I load the extension on dev mode as is suggested [here](https://github.com/reduxjs/redux-devtools/tree/main/extension#installation), the DevTools is not working, so I've tried to change [this line](https://github.com/reduxjs/redux-devtools/blob/main/extension/webpack/base.config.js#L10) as you suggested, but I have no luck with that
